src/extension/tags.c

```diff
@@ -29,9 +30,13 @@
 #define DD_TAG_HTTP_RH_CONTENT_ENCODING "http.response.headers.content-encoding"
 #define DD_TAG_HTTP_RH_CONTENT_LANGUAGE "http.response.headers.content-language"
 #define DD_TAG_HTTP_CLIENT_IP "http.client_ip"
+#define DD_TAG_USER_ID "usr.id"
```

Comment:
Is it missing the `e` in purpose?

Reply:
Yes the tag is `usr.id` for some odd reason.